boat-maven-plugin/src/main/java/com/backbase/oss/boat/GenerateMojo.java

```diff
@@ -81,7 +81,7 @@ public class GenerateMojo extends InputMavenArtifactMojo {
 
     private static String trimCSV(String text) {
         if (isNotEmpty(text)) {
-            return stream(text.split("[,;\\s]+"))
+            return stream(text.split("[,;]+"))
                 .map(StringUtils::trimToNull)
```

Comment:
Could be worth extending this test to that [oas-examples/petstore.yaml ](https://github.com/Backbase/backbase-openapi-tools/blob/9ab36edf1f97a27c4e131860856313981d67c01a/boat-maven-plugin/src/test/resources/oas-examples/petstore.yaml#L55) contains tags with white space? 
 https://github.com/Backbase/backbase-openapi-tools/blob/9ab36edf1f97a27c4e131860856313981d67c01a/boat-maven-plugin/src/test/java/com/backbase/oss/boat/GenerateMojoTests.java#L97

Reply:
I've had a quick look at that test file and I'm not really sure how best to test that the tags are properly split, to be honest.

@bartveenstra, would welcome any pointers you may have?